includes/settings.example.php

```diff
@@ -14,6 +14,7 @@
 	$frommail = '';
 	$askforemail = TRUE;
 	$requireemail = FALSE;
+	$indentation = '    '; // four spaces
```

Comment:
I appreciate the proper four-space indentation, but why do we need to make it configurable ? On the one hand users of pastt will have to change their settings when updating. On the other hand the indentation by four spaces is the defaut setting in Android studio. So why not adding the indentation without the option to change it ?

Reply:
I would propose to propose a four space indentation. 

In translation.php we should add this kind of check *

if ( ! isset($indentation) ) {
    $indentation = '';
}

So we would stick to the old setting, if the indentation is not (yet) defined. This would mean that users can update without breaking their instance of pastt. 

* attention: php written by a Python hero